VideoWeb/VideoWeb/ClientApp/src/app/shared/footer/footer.component.ts

```diff
@@ -14,12 +14,10 @@ export class FooterComponent implements OnInit {
     hideContactUsLink = false;
     privacyPolicyUri = pageUrls.PrivacyPolicy;
     accessibilityUri = pageUrls.Accessibility;
-    hideLinksForUnsupportedBrowser = false;
 
     constructor(private router: Router, private translate: TranslateService, private logger: Logger) {
         this.router.events.pipe(filter((event: RouterEvent) => event instanceof NavigationEnd)).subscribe(x => {
```

Comment:
Not a serious issue since the footer will always be there but we should probably still unsubscribe onDestroy.

Reply:
Added on last commit